SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/PartitionManager.cpp

```diff
@@ -351,15 +523,13 @@ TArray<ComponentData> FPartitionManager::Impl::CreatePartitionEntityComponents(c
 	return Components_Owning;
 }
 
-FPartitionManager::~FPartitionManager() = default;
-
-FPartitionManager::FPartitionManager(const FSubView& InServerWorkerView, Worker_EntityId InStrategyWorkerEntityId,
-									 ViewCoordinator& Coordinator, TUniquePtr<InterestFactory>&& InterestF)
-	: m_Impl(MakeUnique<Impl>(InServerWorkerView, Coordinator, MoveTemp(InterestF)))
+FPartitionManager::FPartitionManager(const FSubView& InServerWorkerView, ViewCoordinator& Coordinator, InterestFactory& InterestF)
+	: m_Impl(MakeUnique<Impl>(InServerWorkerView, Coordinator, InterestF))
 {
-	m_Impl->StrategyWorkerEntityId = InStrategyWorkerEntityId;
 }
 
+FPartitionManager::~FPartitionManager() = default;
```

Comment:
Out of interested, to people we explicitly set constructors to default?

Reply:
Yeah just good practice to be explicit with the intended purpose of the destructor